core/lib/components/os/balenaos.js

```diff
@@ -178,6 +178,13 @@ module.exports = class BalenaOS {
 			case 'jetson-nano':
 				this.bootPartition = 12;
 				break;
+			case 'jetson-xavier':
+				this.bootPartition = 37;
+				break;
+			case 'jetson-xavier-nx-devkit':
```

Comment:
@acostach did you miss pasting the value here?

Reply:
@rcooke-warwick no, both the NX SD and NX eMMC have index 9